cwms-data-api/src/main/java/cwms/cda/data/dao/JooqDao.java

```diff
@@ -69,6 +68,8 @@
 public abstract class JooqDao<T> extends Dao<T> {
     protected static final int ORACLE_CURSOR_TYPE = -10;
     private static final FluentLogger logger = FluentLogger.forEnclosingClass();
+    public static final int FETCH_SIZE = 1000;
+    public static final int HALF_FETCH_SIZE = FETCH_SIZE / 2;
```

Comment:
I wouldnt code this as a half fetch size but instead as Mike called out above. SMALL and set it to 500. I think the relationship between the two values in the code is arbitrary.